package.json

```
@@ -27,7 +27,10 @@
     "next-seo": "^4.26.0",
     "preact": "^10.5.14",
     "react": "17.0.2",
-    "react-dom": "17.0.2"
+    "react-dom": "17.0.2",
+    "react-virtual": "^2.8.0",
+    "react-virtualized-auto-sizer": "^1.0.5",
+    "react-window": "^1.8.6"
```

```suggestion
    "react-virtual": "^2.8.0"
```

You will be removing the dependency other libraries once you think this PR is ready, won't you? 😁

---

Ah, how careless I am :(
Thank you for the correction, Mas